src/modules/auth/containers/screens.scss

```diff
@@ -29,3 +29,13 @@
 .card-footer-button-flex {
 	flex: 1 !important
 }
+
+.unfocused-user-input{
```

Comment:
@queuing4oranges Should be named something like `disabled-username` or `disabled-username-input`